02-GitHub.Rmd

```diff
@@ -9,67 +9,74 @@ opts_chunk$set(tidy.opts=list(width.cutoff=50), tidy = FALSE)
+A folder on your computer that is tracked by Git is called a `repository` or `repo` for short. The main difference between a Git repo and a regular folder on your computer is that a Git repo has a folder within it named `.git`. It's within this `.git` folder that all the changes in the main folder are tracked. Other than that, a Git repo is just a folder on your computer like any other, but with new and wonderful powers.
 
 ## GitHub {#Git2.2}
 
-*GitHub* is an internet service that provides online cloud hosting for .git repos, as well as useful tools for sharing and collaboration. You don't have to have a GitHub account to use .git, and you don't even have to have git installed on your computer to use GitHub. But if you want a cloud backup, want to sync your work to multiple computer, or especially if you want to collaborate or share your work with others (or copy their work!), you'll want to use both .git and GitHub.
+*GitHub* is an internet service that provides online cloud hosting for Git repos, as well as useful tools for sharing and collaboration. You don't have to have a GitHub account to use Git, and you don't have to have Git installed on your computer to use GitHub. But if you want a cloud backup, want to sync your work to multiple computers, or especially if you want to collaborate or share your work with others (or use/contribute to their work!), you'll want to use both Git and GitHub.
 
-> .git tracks all changes within a particular folder (aka git repo) on your computer
+> **Git** tracks all changes within a particular folder (aka git repo) on your computer
 
-> GitHub offers a way to backup or sync your .git repos, or to collaborate with others
+> **GitHub** offers a way to backup or sync your Git repos, or to collaborate with others
 
-If you're a fancy pants programmer you may want to use git from the command line (e.g., terminal or bash depending on your situation) so you can look cool, and admittedly have more fine-grained control over the process. But if you're like the rest of us, and if you're reading this document right now, you want a more user- and beginner-friendly option.
+If you're a fancy pants programmer you may want to use git from the command line (e.g., terminal or bash depending on your situation) so you can look cool, and admittedly have more fine-grained control over the process. But if you're like the rest of us, and if you're reading this document right now, you may want a more user- and beginner-friendly option.
 
 ```{r image1, echo=FALSE, out.width="700px", fig.align='center'}
 knitr::include_graphics("images/Git0.PNG")
 ```
 
 Image credit: devmountain.com
 
-## .git the language {#Git2.3}
```

Comment:
This sounded like you were referring to language like a programming language (might just be my own bias, though)